README.md

```diff
@@ -36,3 +36,5 @@ If you use the examples or code, please cite our article at JOSS in your publish
 ### Requirements
 Julia version 1.6+ 
 
+### Getting Started 
+![eki-getting-started](https://github.com/CliMA/EnsembleKalmanProcesses.jl/assets/45243236/5baf2de9-d55e-41f4-bda9-82f416900bb3)
```

Comment:
Review of figure:
General note: 
- The language in different grey boxes is a bit inconsistent (
- Capitalization/Punctuation is inconsistent
- Maybe we should emphasize using the (more searchable) names we have in the code e.g. `TransformInversion` rather than `ETKI` or `Localizers` rather than `localization`

"Which Methods..."
- "Simply" -> Use EKI: `Inversion()`
- did you wish to include the code-name i.e Sparse EKI is `SparseInversion`, UKI is `Unscented`, ETKI is `TransformInversion`
- ETKI is for high-dimensional data
- I would remove the "Note: " as these are typically included in the description in the tool on the docs, and the Note's that are there are currently inconsistent between tools
- It's weird having CES branching with arrows from EKS/UKI - though I'm not too fussed (maybe just have a dotted line, or have a disjointed box?

Additional features:
- "Try using SampleSuccGauss" -> "Try using our FailureHandler SampleSuccGauss"
- "if you cannot..." box just replace with "Try using our Localizers"

Which Priors should I choose:
- Maybe just make the grey boxes read a little more consistently
- "only have known constraints" - remove word "only"
- We should impress that we want people to use the constrained_gaussian constructor in most cases, as it's easy to use!

Is the method working?
- say we have a troubleshooting page
- parallelization is spelled incorrectly

Reply:
Thanks for the feedback and discussion. Updated figure is now in the branch.